applicationconfig/client/src/samples/java/ConfigurationSets.java

```diff
@@ -0,0 +1,126 @@
+import com.azure.applicationconfig.ConfigurationClient;
+import com.azure.applicationconfig.ConfigurationClientCredentials;
+import com.azure.applicationconfig.models.ConfigurationSetting;
+import com.azure.applicationconfig.models.SettingSelector;
+import com.azure.common.annotations.Beta;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+/**
+ * Sample demonstrates how to use Azure Application Configuration to switch between "beta" and "production"
+ * configuration sets.
+ */
+public class ConfigurationSets {
+    private static final String CONNECTION_STRING_KEY = "connection-string";
+    private static final String KEY_VAULT_KEY = "key-vault";
+    private static final String BETA = "beta";
+    private static final String PRODUCTION = "production";
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    public static void main(String[] args) throws NoSuchAlgorithmException, InvalidKeyException, IOException {
+        // Retrieve the connection string from the configuration store.
+        // You can get the string from your Azure portal.
+        String connectionString = "Endpoint=https://azconfig-conniey.azconfig.io;Id=0-l1-s0:AEZrHeRv1w3fJ/2ge29R;Secret=2ZIPIs1kudLe7ZzfaD5UZdZotXJysw+Zs/Uc6GbWmqA=";
+
+        // Instantiate a client that will be used to call the service.
+        ConfigurationClient client = ConfigurationClient.builder()
+                .credentials(new ConfigurationClientCredentials(connectionString))
+                .build();
+
+        // Creating two sets of configuration, one for beta testing and one for production when we are ready to ship.
+        KeyVaultConfiguration betaKeyVault = new KeyVaultConfiguration().endpointUri("https://beta-keyvault.vault.azure.net").secret("beta_secret");
+        KeyVaultConfiguration productionKeyVault = new KeyVaultConfiguration().endpointUri("https://production-keyvault.vault.azure.net").secret("production_secret");
+
+        // Adding both of those configuration sets to Azure App Configuration.
+        addConfigurations(client, BETA, "https://beta-storage.core.windows.net", betaKeyVault, false).block();
```

Comment:
If you want to leave `block()` calls in, I would suggest adding a comment explaining that this was done to prevent execution continuing before it was completed.

Reply:
I've added comments about this.